Tests/KlaviyoCoreTests/__Snapshots__/EncodableTests/testEventPayloadWithoutMetadata.1.json

```diff
@@ -21,23 +21,11 @@
         }
       },
       "properties" : {
-        "App Build" : "1",
-        "App Name" : "FooApp",
-        "App Version" : "1.2.3",
-        "Application ID" : "com.klaviyo.fooapp",
-        "blob" : "blob",
-        "Device Manufacturer" : "Orange",
-        "Device Model" : "jPhone 1,1",
-        "hello" : {
```

Comment:
Yah something might be wrong here...you are missing custom properties too..

Reply:
yeah looks like the old event that was being used had them but I don't seem to be passing that custom properties to the new initializer. WIll add it in.

Reply:
Added in properties to events. Now it looks like this, 

```json
{
  "data" : {
    "attributes" : {
      "metric" : {
        "data" : {
          "attributes" : {
            "name" : "test"
          },
          "type" : "metric"
        }
      },
      "profile" : {
        "data" : {
          "attributes" : {
            "anonymous_id" : "anon-id",
            "properties" : {

            }
          },
          "type" : "profile"
        }
      },
      "properties" : {
        "App Build" : "1",
        "App ID" : "com.klaviyo.fooapp",
        "App Name" : "FooApp",
        "App Version" : "1.2.3",
        "blob" : "blob",
        "Device ID" : "fe-fi-fo-fum",
        "Device Manufacturer" : "Orange",
        "Device Model" : "jPhone 1,1",
        "hello" : {
          "sub" : "dict"
        },
        "OS Name" : "iOS",
        "OS Version" : "1.1.1",
        "Push Token" : "",
        "SDK Name" : "swift",
        "SDK Version" : "3.2.0",
        "stuff" : 2
      },
      "time" : "2009-02-13T23:31:30Z",
      "unique_id" : "00000000-0000-0000-0000-000000000001"
    },
    "type" : "event"
  }
}

```